DuckDuckGo/NetworkProtectionStatusView.swift

```diff
@@ -40,9 +40,12 @@ struct NetworkProtectionStatusView: View {
             }
             settings()
         }
-        .animation(.default, value: statusModel.shouldShowError)
         .padding(.top, statusModel.error == nil ? 0 : -20)
-        .animation(.default, value: statusModel.shouldShowConnectionDetails)
+        .if(statusModel.animationsOn, transform: {
```

Comment:
The animation bug this fixes became a lot more noticeable with the addition of the new section. So I fixed it here.